LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -33,11 +33,11 @@ async function dispatchMouseActions(actions, pointerType)
         switch (action.type) {
         case "pointerMove":
             const origin = { x: 0, y: 0 };
-            if (action.origin instanceof Element) {
+            if (action.origin instanceof action.origin?.ownerDocument.defaultView?.Element) {
```

Comment:
```suggestion
            if (action.origin instanceof action.origin?.ownerDocument?.defaultView?.Element) {
```

The origin can be a string, so we should assume ownerDocument can be null too.